website/docs/commands/misc/config.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Config
+sidebar_position: 1
+---
+
+The `config` sub-command allows to get and set various configuration values, used by
+other Scala CLI sub-commands.
+
+Examples of use:
+```text
```

Comment:
```suggestion
```bash
imo it's better to use `bash` code blocks here, as it allows for some syntax coloring.
command outputs can then be put as comments in the code block.
this applies to the other code blocks in this doc as well.